src/vs/workbench/test/node/api/extHostLanguageFeatures.test.ts

```diff
@@ -567,7 +567,7 @@ suite('ExtHostLanguageFeatures', function () {
 
```

Comment:
I am struggling to get tests working and debugable. I opened an issue with vscode to see if they can help.